palamedes/__init__.py

```diff
@@ -1,16 +1,152 @@
-from Bio.Align import PairwiseAligner
+from Bio.Align import PairwiseAligner, Alignment
 from Bio.SeqRecord import SeqRecord
 from hgvs.sequencevariant import SequenceVariant
 
-from palamedes.config import MOLECULE_TYPE_PROTEIN, ALT_SEQUENCE_ID, REF_SEQUENCE_ID
-from palamedes.align import generate_seq_record, generate_alignment, generate_variant_blocks
+from palamedes.align import generate_seq_record, generate_variant_blocks, reverse_seq_record
 from palamedes.hgvs.utils import categorize_variant_block
 from palamedes.hgvs.builders import BUILDER_CONFIG
-
+from palamedes.config import (
+    DEFAULT_EXTEND_GAP_SCORE,
+    DEFAULT_MATCH_SCORE,
+    DEFAULT_MISMATCH_SCORE,
+    DEFAULT_OPEN_GAP_SCORE,
+    GLOBAL_ALIGN_MODE,
+    MOLECULE_TYPE_PROTEIN,
+     ALT_SEQUENCE_ID,
+     REF_SEQUENCE_ID,
+)
 
 __version__ = "0.0.8"
 
 
+def generate_hgvs_variants_from_alignment(
+    alignment: Alignment, use_non_standard_substitution_rules: bool = False, molecule_type: str = MOLECULE_TYPE_PROTEIN
+) -> list[SequenceVariant]:
+    """
```

Comment:
Adding some examples here using the follow format might be nice for the API docs:
```
    .. code-block:: python

        >>> from palamedes import generate_hgvs_variants
        >>> generate_hgvs_variants("PFKISIHL", "TPFKISIH")
        [
            SequenceVariant(ac=ref, type=p, posedit=Pro1extThr-1, gene=None),
            SequenceVariant(ac=ref, type=p, posedit=Leu8del, gene=None),
        ]
```
This is from the doc string for `generate_hgvs_variants`